packages/eslint-plugin/docs/rules/README.md

```diff
@@ -148,12 +151,13 @@ In these cases, we create what we call an extension rule; a rule within our plug
 | [`@typescript-eslint/no-useless-constructor`](./no-useless-constructor.md)                   | Disallow unnecessary constructors                                                    |                    |          |                   |
 | [`@typescript-eslint/object-curly-spacing`](./object-curly-spacing.md)                       | Enforce consistent spacing inside braces                                             |                    | :wrench: |                   |
-| [`@typescript-eslint/padding-line-between-statements`](./padding-line-between-statements.md) | require or disallow padding lines between statements                                 |                    | :wrench: |                   |
+| [`@typescript-eslint/padding-line-between-statements`](./padding-line-between-statements.md) | Require or disallow padding lines between statements                                 |                    | :wrench: |                   |
 | [`@typescript-eslint/quotes`](./quotes.md)                                                   | Enforce the consistent use of either backticks, double, or single quotes             |                    | :wrench: |                   |
 | [`@typescript-eslint/require-await`](./require-await.md)                                     | Disallow async functions which have no `await` expression                            | :white_check_mark: |          | :thought_balloon: |
-| [`@typescript-eslint/return-await`](./return-await.md)                                       | Enforces consistent returning of awaited values                                      |                    | :wrench: | :thought_balloon: |
+| [`@typescript-eslint/return-await`](./return-await.md)                                       | Enforce consistent returning of awaited values                                       |                    | :wrench: | :thought_balloon: |
 | [`@typescript-eslint/semi`](./semi.md)                                                       | Require or disallow semicolons instead of ASI                                        |                    | :wrench: |                   |
-| [`@typescript-eslint/space-before-function-paren`](./space-before-function-paren.md)         | Enforces consistent spacing before function parenthesis                              |                    | :wrench: |                   |
-| [`@typescript-eslint/space-infix-ops`](./space-infix-ops.md)                                 | This rule is aimed at ensuring there are spaces around infix operators.              |                    | :wrench: |                   |
+| [`@typescript-eslint/space-before-blocks`](./space-before-blocks.md)                         | Enforce consistent spacing before blocks                                             |                    | :wrench: |                   |
```

Comment:
There are some rules missing in this file. I assume it's simply an oversight.

Reply:
there shouldn't be any missing files - it should be tested against and enforced.

Reply:
I don't think it's tested as comprehensively as the main readme, no.

Reply:
We should definitely fix this! Considering this file powers our website - it should be generated and tested just like the one in the main readme!

Or perhaps we should just generate the website table at website build time instead of making it rely on a markdown file on disk?

Reply:
Certainly. I've added `docs/rules/README.md` to the tests.

> we should just generate the website table at website build time

We can do that later; it's a bit harder.